docker/manager.go

```diff
@@ -31,10 +31,10 @@ import (
 
 // Manager manages all docker operations, like build, push, etc.
 type Manager struct {
-	client     *docker_client.Client
-	registry   string
-	authConfig *AuthConfig
-	endPoint   string
+	Client     *docker_client.Client
+	Registry   string
+	AuthConfig *AuthConfig
+	EndPoint   string
```

Comment:
Why expose these vars? We have functions to get them, such as: GetDockerAuthConfig, GetDockerRegistry. If you expose them, them can be modified